lib/membrane_rtmp_plugin/rtmp/source/source_bin.ex

```diff
@@ -39,14 +39,22 @@ defmodule Membrane.RTMP.SourceBin do
                 An URL on which the client is expected to connect, for example:
                 rtmp://127.0.0.1:1935/app/stream_key
                 """
+              ],
+              client_timeout: [
+                default: Membrane.Time.seconds(5),
+                spec: Membrane.Time.t(),
+                description: """
+                Time after which an unused client connection is automatically closed. Defaults to 5 seconds.
```

Comment:
I would add an explicit information that it is expressed in Membrane.Time.t(), not in millis